css/styles.css

```diff
@@ -1,63 +1,58 @@
-body{
+.container{
     width: 1280px;
-    height: 1626px;
+    margin: auto;
+    font-family: Roboto, sans-serif;
+}
+.container_nav{
     padding-top: 32px;
-    background-color: white;
 }
+
```

Comment:
Better to use the next properties for the header navigation
https://stackblitz.com/edit/js-aqlfya?file=index.js,index.html,style.css